app/schemas/get-manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id" : "http://e-manifest.18f.gov/schemas/get-manifest.json",
+  "method": "GET",
+  "title": "e-Manifest API - GET Manifest",
+  "type": [
+    "object"
+  ],
+  "required": [
+    "document_id",
+    "activity_id",
+    "content"
+  ],
+  "definitions" : {
+    "content" : {
+      "$ref" : "http://e-manifest.18f.gov/schemas/post-manifest.json#"
```

Comment:
what about naming the `content` file something like `manifest-content.json` and then including it in `get-manifest` and `post-manifest` ? Not sure how duplicate keys (eg: `method`) work when we ref like this.